.gitignore

```diff
@@ -1,2 +1,2 @@
 node_modules/
-auth.json
\ No newline at end of file
+thanks-bot/auth.json
```

Comment:
This line excludes `auth.json` in `thanks-bot`, but includes `auth.json` in `butler-bot`.
Sorry for the confusion, let's leave it just `auth.json`.